--- a/katze/midori-uri.vala
+++ b/katze/midori-uri.vala
@@ -114,7 +114,7 @@
             string escaped = GLib.Uri.escape_string (keywords, ":/", true);
             /* Allow DuckDuckGo to distinguish Midori and in turn share revenue */
             if (uri == "https://duckduckgo.com/?q=%s")
-                return "https://duckduckgo.com/?q=%s&t=midori".printf (escaped);
+                return "https://duckduckgo.com/?q=%s&t=elemos".printf (escaped);
             if (uri.str ("%s") != null)
                 return uri.printf (escaped);
             return uri + escaped;
